lib/tasks/build-components.js

```diff
@@ -1,14 +1,29 @@
 'use strict'
 
 const paths = require('../../config/paths.json')
+const packageJson = require('../../package.json')
+
 const gulp = require('gulp')
+const rename = require('gulp-rename')
+
+const transpiler = require('../transpilation/transpiler.js')
+
+let transpileRunner = templateLanguage => {
+  return gulp.src(paths.appComponents + '*/*.nunj')
+    // `*/* vs `**/*` avoids including _preview.nuj in /components, which isn't a components
+    .pipe(transpiler.transpileComponent(templateLanguage, packageJson.version))
+    // no .html prefix like build:template, components have different naming convention
+    // @FIXME: standardise the naming convention in source
+    // `dirname` drops the extra level of dir, `button/button.ext` => `button.ext`
+    .pipe(rename({dirname: '', extname: '.' + templateLanguage}))
+    .pipe(gulp.dest(paths.bundleComponents))
+}
 
-// Task for building the components
+// Task for transpiling the templates
```

Comment:
Task name is still `build:components`. If this comment change is correct then do we want to rename the task?